src/app/components/Header/Header.tsx

```diff
@@ -26,6 +26,21 @@ const specialContent: SpecialContent = {
       timely public health action using CDC's free, cloud-based products built
       from Data Integration Building Blocks, or DIBBs.`,
   },
+  '/our-products': {
+    heroClass: 'homepage-hero',
+    heroHeader: `Our ecosystem of DIBBs products`,
+    heroSubheader: `Find out how DIBBs products can help empower your jurisdiction with more usable data.`,
+  },
+  '/case-studies': {
+    heroClass: 'homepage-hero',
```

Comment:
I noticed that the image used by the hero component is different in the designs depending on which page you're looking at. Is this something we want to tackle now or return to later?

Reply:
I didn't realize they were different in the mockups....fixed!